icaruscode/Filters/FilterCRTPMTMatching_module.cc

```diff
@@ -187,13 +185,33 @@ icarus::crt::CRTMatches CRTHitmatched(
 }
 
 class icarus::crt::FilterCRTPMTMatching : public art::EDFilter {
+
+/**
+ * @brief Rejects events with only incoming particles in time with the beam.
+ *
+ * This filtering module is based on CRT the CRT PMT matching.
+ * It starts by considering only the flashes which are within the beam gate
+ * and it tries to match them with one or more CRT Hits using a configurable
+ * time interval, which, at the time (19/04/2023) is chosen as +/-100 ns.
+ * If the flash is matched, the relative time will determine if the mu candidate
+ * is entering or exiting the TPC. Depending on the amound and relative time of 
+ * the match, a classification is provided to the flash.
+ * The filtering module has 3 options: loose (everything goes through), medium
+ * (removes events where the flash is matched with one Top CRT hit before the
+ * flash) and tight (removes all the events where the flashes are matched with
+ * CRT hits before the Flash).
+ * If multiple flashes are in the beam gate, the filtering logic applies to the 
+ * AND of the flashes classification.
+ * For questions and maintenance ask Francesco Poppi.
+ */
+
  public:
   using CRTHit = sbn::crt::CRTHit;
 
   explicit FilterCRTPMTMatching(fhicl::ParameterSet const& p);
-
   // Required functions.
-  void getTriggerConf(art::Run&);
+  //void getTriggerConf(art::Run const&);
+  void beginRun(art::Run const& run);
```

Comment:
Mark `beginRun()` `override`, and I think you need to remove `const` (with `override`, GCC will tell you if that's the case: `"override" overrides nothing` or something like that).
```suggestion
  void beginRun(art::Run& run) override;
```